From 6b7dc84f278f197ecad202106556a233f1a68170 Mon Sep 17 00:00:00 2001
From: Marek Vasut <marex@denx.de>
Date: Sat, 11 Apr 2020 05:02:46 +0200
Subject: [PATCH 25/52] ARM: dts: stm32: Fill GPIO line names on AV96

Fill in the custom GPIO line names used by DH.

Upstream-Status: Posted [https://patchwork.kernel.org/patch/11682845/]
Signed-off-by: Marek Vasut <marex@denx.de>
Cc: Alexandre Torgue <alexandre.torgue@st.com>
Cc: Maxime Coquelin <mcoquelin.stm32@gmail.com>
Cc: Patrice Chotard <patrice.chotard@st.com>
Cc: Patrick Delaunay <patrick.delaunay@st.com>
Cc: linux-stm32@st-md-mailman.stormreply.com
To: linux-arm-kernel@lists.infradead.org
---
 .../boot/dts/stm32mp15xx-dhcor-avenger96.dtsi | 35 +++++++++++++++++++
 1 file changed, 35 insertions(+)

diff --git a/arch/arm/boot/dts/stm32mp15xx-dhcor-avenger96.dtsi b/arch/arm/boot/dts/stm32mp15xx-dhcor-avenger96.dtsi
index ec02cee1dd9b..6df0ed9a73c8 100644
--- a/arch/arm/boot/dts/stm32mp15xx-dhcor-avenger96.dtsi
+++ b/arch/arm/boot/dts/stm32mp15xx-dhcor-avenger96.dtsi
@@ -162,6 +162,41 @@ phy0: ethernet-phy@7 {
 	};
 };
 
+&gpioa {
+	gpio-line-names = "", "", "", "",
+			  "", "", "", "",
+			  "", "", "", "DH-GPIO-K",
+			  "DH-GPIO-I", "", "DH-GPIO-A", "";
+};
+
+&gpiob {
+	gpio-line-names = "", "", "", "",
+			  "", "DH-GPIO-J", "", "",
+			  "", "", "", "DH-GPIO-B",
+			  "", "DH-GPIO-L", "", "";
+};
+
+&gpioc {
+	gpio-line-names = "", "", "", "DH-GPIO-C",
+			  "", "", "", "",
+			  "", "", "", "",
+			  "", "", "", "";
+};
+
+&gpiod {
+	gpio-line-names = "", "", "", "",
+			  "", "", "", "",
+			  "DH-GPIO-D", "", "", "",
+			  "", "", "DH-GPIO-E", "DH-GPIO-F";
+};
+
+&gpiof {
+	gpio-line-names = "", "", "", "",
+			  "", "", "", "",
+			  "", "", "", "",
+			  "DH-GPIO-G", "DH-GPIO-H", "", "";
+};
+
 &i2c1 {	/* X6 I2C1 */
 	pinctrl-names = "default";
 	pinctrl-0 = <&i2c1_pins_b>;
-- 
2.29.2

